Start testing: Feb 07 23:28 +06
----------------------------------------------------------
1/11 Testing: incorrect_solution_tests
1/11 Test: incorrect_solution_tests
Command: "/Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/304044/build/incorrect_solution_tests"
Directory: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/304044/build
"incorrect_solution_tests" start time: Feb 07 23:28 +06
Output:
----------------------------------------------------------
Running main() from /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/304044/build/_deps/googletest-src/googletest/src/gtest_main.cc
[==========] Running 1 test from 1 test suite.
[----------] Global test environment set-up.
[----------] 1 test from PermutationFactoryTest
[ RUN      ] PermutationFactoryTest.HandlesTestCases
/Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/304044/test.cpp:95: Failure
Expected equality of these values:
  result[i].size()
    Which is: 2
  expected[i].size()
    Which is: 4
Mismatch in number of operations for test case 4

[  FAILED  ] PermutationFactoryTest.HandlesTestCases (0 ms)
[----------] 1 test from PermutationFactoryTest (0 ms total)

[----------] Global test environment tear-down
[==========] 1 test from 1 test suite ran. (0 ms total)
[  PASSED  ] 0 tests.
[  FAILED  ] 1 test, listed below:
[  FAILED  ] PermutationFactoryTest.HandlesTestCases

 1 FAILED TEST
<end of output>
Test time =   0.02 sec
----------------------------------------------------------
Test Failed.
"incorrect_solution_tests" end time: Feb 07 23:28 +06
"incorrect_solution_tests" time elapsed: 00:00:00
----------------------------------------------------------

Unable to find executable: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/304044/build/model1_tests
2/11 Testing: model1_tests
2/11 Test: model1_tests
Command: ""
Directory: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/304044/build
"model1_tests" start time: 
Output:
----------------------------------------------------------
<end of output>
Test time =   0.00 sec
----------------------------------------------------------
Test Passed.
"model1_tests" end time: Feb 07 23:28 +06
"model1_tests" time elapsed: 00:00:00
----------------------------------------------------------

3/11 Testing: model10_tests
3/11 Test: model10_tests
Command: "/Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/304044/build/model10_tests"
Directory: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/304044/build
"model10_tests" start time: Feb 07 23:28 +06
Output:
----------------------------------------------------------
Running main() from /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/304044/build/_deps/googletest-src/googletest/src/gtest_main.cc
[==========] Running 1 test from 1 test suite.
[----------] Global test environment set-up.
[----------] 1 test from PermutationFactoryTest
[ RUN      ] PermutationFactoryTest.HandlesTestCases
/Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/304044/test.cpp:95: Failure
Expected equality of these values:
  result[i].size()
    Which is: 2
  expected[i].size()
    Which is: 4
Mismatch in number of operations for test case 4

[  FAILED  ] PermutationFactoryTest.HandlesTestCases (0 ms)
[----------] 1 test from PermutationFactoryTest (0 ms total)

[----------] Global test environment tear-down
[==========] 1 test from 1 test suite ran. (0 ms total)
[  PASSED  ] 0 tests.
[  FAILED  ] 1 test, listed below:
[  FAILED  ] PermutationFactoryTest.HandlesTestCases

 1 FAILED TEST
<end of output>
Test time =   0.02 sec
----------------------------------------------------------
Test Failed.
"model10_tests" end time: Feb 07 23:28 +06
"model10_tests" time elapsed: 00:00:00
----------------------------------------------------------

4/11 Testing: model2_tests
4/11 Test: model2_tests
Command: "/Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/304044/build/model2_tests"
Directory: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/304044/build
"model2_tests" start time: Feb 07 23:28 +06
Output:
----------------------------------------------------------
Running main() from /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/304044/build/_deps/googletest-src/googletest/src/gtest_main.cc
[==========] Running 1 test from 1 test suite.
[----------] Global test environment set-up.
[----------] 1 test from PermutationFactoryTest
[ RUN      ] PermutationFactoryTest.HandlesTestCases
/Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/304044/test.cpp:95: Failure
Expected equality of these values:
  result[i].size()
    Which is: 2
  expected[i].size()
    Which is: 4
Mismatch in number of operations for test case 4

[  FAILED  ] PermutationFactoryTest.HandlesTestCases (0 ms)
[----------] 1 test from PermutationFactoryTest (0 ms total)

[----------] Global test environment tear-down
[==========] 1 test from 1 test suite ran. (0 ms total)
[  PASSED  ] 0 tests.
[  FAILED  ] 1 test, listed below:
[  FAILED  ] PermutationFactoryTest.HandlesTestCases

 1 FAILED TEST
<end of output>
Test time =   0.02 sec
----------------------------------------------------------
Test Failed.
"model2_tests" end time: Feb 07 23:28 +06
"model2_tests" time elapsed: 00:00:00
----------------------------------------------------------

5/11 Testing: model3_tests
5/11 Test: model3_tests
Command: "/Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/304044/build/model3_tests"
Directory: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/304044/build
"model3_tests" start time: Feb 07 23:28 +06
Output:
----------------------------------------------------------
Running main() from /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/304044/build/_deps/googletest-src/googletest/src/gtest_main.cc
[==========] Running 1 test from 1 test suite.
[----------] Global test environment set-up.
[----------] 1 test from PermutationFactoryTest
[ RUN      ] PermutationFactoryTest.HandlesTestCases
/Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/304044/test.cpp:95: Failure
Expected equality of these values:
  result[i].size()
    Which is: 2
  expected[i].size()
    Which is: 4
Mismatch in number of operations for test case 4

[  FAILED  ] PermutationFactoryTest.HandlesTestCases (0 ms)
[----------] 1 test from PermutationFactoryTest (0 ms total)

[----------] Global test environment tear-down
[==========] 1 test from 1 test suite ran. (0 ms total)
[  PASSED  ] 0 tests.
[  FAILED  ] 1 test, listed below:
[  FAILED  ] PermutationFactoryTest.HandlesTestCases

 1 FAILED TEST
<end of output>
Test time =   0.02 sec
----------------------------------------------------------
Test Failed.
"model3_tests" end time: Feb 07 23:28 +06
"model3_tests" time elapsed: 00:00:00
----------------------------------------------------------

Unable to find executable: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/304044/build/model4_tests
6/11 Testing: model4_tests
6/11 Test: model4_tests
Command: ""
Directory: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/304044/build
"model4_tests" start time: 
Output:
----------------------------------------------------------
<end of output>
Test time =   0.00 sec
----------------------------------------------------------
Test Passed.
"model4_tests" end time: Feb 07 23:28 +06
"model4_tests" time elapsed: 00:00:00
----------------------------------------------------------

Unable to find executable: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/304044/build/model5_tests
7/11 Testing: model5_tests
7/11 Test: model5_tests
Command: ""
Directory: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/304044/build
"model5_tests" start time: 
Output:
----------------------------------------------------------
<end of output>
Test time =   0.00 sec
----------------------------------------------------------
Test Passed.
"model5_tests" end time: Feb 07 23:28 +06
"model5_tests" time elapsed: 00:00:00
----------------------------------------------------------

8/11 Testing: model6_tests
8/11 Test: model6_tests
Command: "/Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/304044/build/model6_tests"
Directory: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/304044/build
"model6_tests" start time: Feb 07 23:28 +06
Output:
----------------------------------------------------------
Running main() from /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/304044/build/_deps/googletest-src/googletest/src/gtest_main.cc
[==========] Running 1 test from 1 test suite.
[----------] Global test environment set-up.
[----------] 1 test from PermutationFactoryTest
[ RUN      ] PermutationFactoryTest.HandlesTestCases
/Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/304044/test.cpp:95: Failure
Expected equality of these values:
  result[i].size()
    Which is: 2
  expected[i].size()
    Which is: 4
Mismatch in number of operations for test case 4

[  FAILED  ] PermutationFactoryTest.HandlesTestCases (0 ms)
[----------] 1 test from PermutationFactoryTest (0 ms total)

[----------] Global test environment tear-down
[==========] 1 test from 1 test suite ran. (0 ms total)
[  PASSED  ] 0 tests.
[  FAILED  ] 1 test, listed below:
[  FAILED  ] PermutationFactoryTest.HandlesTestCases

 1 FAILED TEST
<end of output>
Test time =   0.02 sec
----------------------------------------------------------
Test Failed.
"model6_tests" end time: Feb 07 23:28 +06
"model6_tests" time elapsed: 00:00:00
----------------------------------------------------------

9/11 Testing: model7_tests
9/11 Test: model7_tests
Command: "/Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/304044/build/model7_tests"
Directory: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/304044/build
"model7_tests" start time: Feb 07 23:28 +06
Output:
----------------------------------------------------------
Running main() from /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/304044/build/_deps/googletest-src/googletest/src/gtest_main.cc
[==========] Running 1 test from 1 test suite.
[----------] Global test environment set-up.
[----------] 1 test from PermutationFactoryTest
[ RUN      ] PermutationFactoryTest.HandlesTestCases
/Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/304044/test.cpp:95: Failure
Expected equality of these values:
  result[i].size()
    Which is: 2
  expected[i].size()
    Which is: 4
Mismatch in number of operations for test case 4

[  FAILED  ] PermutationFactoryTest.HandlesTestCases (0 ms)
[----------] 1 test from PermutationFactoryTest (0 ms total)

[----------] Global test environment tear-down
[==========] 1 test from 1 test suite ran. (0 ms total)
[  PASSED  ] 0 tests.
[  FAILED  ] 1 test, listed below:
[  FAILED  ] PermutationFactoryTest.HandlesTestCases

 1 FAILED TEST
<end of output>
Test time =   0.02 sec
----------------------------------------------------------
Test Failed.
"model7_tests" end time: Feb 07 23:28 +06
"model7_tests" time elapsed: 00:00:00
----------------------------------------------------------

10/11 Testing: model8_tests
10/11 Test: model8_tests
Command: "/Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/304044/build/model8_tests"
Directory: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/304044/build
"model8_tests" start time: Feb 07 23:28 +06
Output:
----------------------------------------------------------
Running main() from /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/304044/build/_deps/googletest-src/googletest/src/gtest_main.cc
[==========] Running 1 test from 1 test suite.
[----------] Global test environment set-up.
[----------] 1 test from PermutationFactoryTest
[ RUN      ] PermutationFactoryTest.HandlesTestCases
/Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/304044/test.cpp:95: Failure
Expected equality of these values:
  result[i].size()
    Which is: 2
  expected[i].size()
    Which is: 4
Mismatch in number of operations for test case 4

[  FAILED  ] PermutationFactoryTest.HandlesTestCases (0 ms)
[----------] 1 test from PermutationFactoryTest (0 ms total)

[----------] Global test environment tear-down
[==========] 1 test from 1 test suite ran. (0 ms total)
[  PASSED  ] 0 tests.
[  FAILED  ] 1 test, listed below:
[  FAILED  ] PermutationFactoryTest.HandlesTestCases

 1 FAILED TEST
<end of output>
Test time =   0.02 sec
----------------------------------------------------------
Test Failed.
"model8_tests" end time: Feb 07 23:28 +06
"model8_tests" time elapsed: 00:00:00
----------------------------------------------------------

Unable to find executable: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/304044/build/model9_tests
11/11 Testing: model9_tests
11/11 Test: model9_tests
Command: ""
Directory: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/304044/build
"model9_tests" start time: 
Output:
----------------------------------------------------------
<end of output>
Test time =   0.00 sec
----------------------------------------------------------
Test Passed.
"model9_tests" end time: Feb 07 23:28 +06
"model9_tests" time elapsed: 00:00:00
----------------------------------------------------------

End testing: Feb 07 23:28 +06
